crates/iota/src/iota_commands.rs

```diff
@@ -217,6 +217,14 @@ pub enum IotaCommand {
         #[clap(long)]
         epoch_duration_ms: Option<u64>,
 
+        /// Make the fullnode dump executed checkpoints as files to this
+        /// directory. This is incompatible with --no-full-node.
+        ///
+        /// If --with-indexer is set, this defaults to a temporary directory.
+        #[cfg(feature = "indexer")]
+        #[clap(long, value_name = "DATA_INGESTION_DIR")]
```

Comment:
```suggestion
        #[arg(long, value_name = "DATA_INGESTION_DIR")]
```

Reply:
Ah, I forgot the other PR wasn't merged, do it or not, does not matter much

Reply:
Makes sense already, thanks